File-Formats/PDF/Convert-HTML-To-PDF/aws.md

```diff
@@ -7,132 +7,127 @@ documentation: UG
+   {% endhighlight %}
 
-//Save the document into stream.
-MemoryStream memoryStream = new MemoryStream();
+6. Add the following code snippet in Function.cs to create a PDF document.
 
-//Save and Close the PDFDocument.
-document.Save(memoryStream);
-document.Close(true);
+   {% highlight c# tabtitle="C#" %}
 
-return Convert.ToBase64String(memoryStream.ToArray());
+            //Initialize HTML to PDF converter with Blink rendering engine.
+            HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+            //Convert URL to PDF.
+            PdfDocument document = htmlConverter.Convert(input);
+            //Save the document into stream.
+            MemoryStream memoryStream = new MemoryStream();
+            //Save and Close the PDF Document.
+            document.Save(memoryStream);
+            document.Close(true);
+            return Convert.ToBase64String(memoryStream.ToArray());
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Right-click the project and select Publish to AWS Lambda. 
+7. Right-click the project and select Publish to AWS Lambda. 
 ![Convert HTMLToPDF AWS Step5](htmlconversion_images/AWS5.png)  
 
-Create a new AWS profile in the Upload Lambda Function Window. After creating the profile, add a name for the Lambda function to publish. Then, click Next.   
+8. Create a new AWS profile in the Upload Lambda Function Window. After creating the profile, add a name for the Lambda function to publish. Then, click Next.   
 ![Convert HTMLToPDF AWS Step6](htmlconversion_images/AWS6.png)     
 
-In the Advanced Function Details window, specify the Role Name as based on AWS Managed policy. After selecting the role, click the Upload button to deploy your application.
+9. In the Advanced Function Details window, specify the Role Name as based on AWS Managed policy. After selecting the role, click the Upload button to deploy your application.
 ![Convert HTMLToPDF AWS Step7](htmlconversion_images/AWS7.png)   
 
-After deploying the application, Sign in your AWS account and you can see the published Lambda function in AWS console. 
+10. After deploying the application, Sign in your AWS account and you can see the published Lambda function in AWS console. 
 ![Convert HTMLToPDF AWS Step8](htmlconversion_images/AWS8.png)
 
 ## Refer to the following steps to invoke the AWS Lambda function from the console application:
 
-Create a new console project.  
+1. Create a new console project.  
 ![Convert HTMLToPDF AWS Step9](htmlconversion_images/AWS9.png)
 
-Create a project name and select folder path. 
+2. Create a project name and select folder path. 
 ![Convert HTMLToPDF AWS Step10](htmlconversion_images/AWS10.png)   
 
-Install the AWSSDK.Core, AWSSDK.Lambda and Newtonsoft.Json package as a reference to your main application from the NuGet.org.    
+3. Install the AWSSDK.Core, AWSSDK.Lambda and Newtonsoft.Json package as a reference to your main application from the NuGet.org.    
```

Comment:
Link all the NuGet package mentioned here including nuget.org

Reply:
added